test/dpul_collections_web/features/locale_test.exs

```diff
@@ -30,4 +31,22 @@ defmodule DpulCollectionsWeb.Features.LocaleTest do
 
     Solr.delete_all(active_collection())
   end
+
+  test "renders facet with states", %{conn: conn} do
```

Comment:
Can we move this test to the search_live_test file?

Reply:
Hmm... @hackartisan that was the only way I could test the conditionals for whether to show or hide the pill element. Is there a better way to test this?

Reply:
Oh, I see what you mean... move it out of the locale test. Yes! It should go in the `features/search_bar_test.exs` file, right?

Reply:
@hackartisan yes, I was able to move it into search_live_test.